Dial down the Rust minimum-version requirement by a couple notches.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -64,3 +64,3 @@ resolver = "2"
 [workspace.package]
-rust-version = "1.90.0"
+rust-version = "1.88.0"
 
--- a/python/mozboot/mozboot/util.py
+++ b/python/mozboot/mozboot/util.py
@@ -15,3 +15,3 @@ from mach.util import get_state_dir
 # Keep in sync with rust-version in top-level Cargo.toml.
-MINIMUM_RUST_VERSION = "1.90.0"
+MINIMUM_RUST_VERSION = "1.88.0"
 
--- a/third_party/rust/mtu/Cargo.toml
+++ b/third_party/rust/mtu/Cargo.toml
@@ -13,3 +13,3 @@
 edition = "2024"
-rust-version = "1.90.0"
+rust-version = "1.88.0"
 name = "mtu"
--- a/third_party/rust/neqo-bin/Cargo.toml
+++ b/third_party/rust/neqo-bin/Cargo.toml
@@ -13,3 +13,3 @@
 edition = "2024"
-rust-version = "1.90.0"
+rust-version = "1.88.0"
 name = "neqo-bin"
--- a/third_party/rust/neqo-common/Cargo.toml
+++ b/third_party/rust/neqo-common/Cargo.toml
@@ -13,3 +13,3 @@
 edition = "2024"
-rust-version = "1.90.0"
+rust-version = "1.88.0"
 name = "neqo-common"
--- a/third_party/rust/neqo-crypto/Cargo.toml
+++ b/third_party/rust/neqo-crypto/Cargo.toml
@@ -13,3 +13,3 @@
 edition = "2024"
-rust-version = "1.90.0"
+rust-version = "1.88.0"
 name = "neqo-crypto"
--- a/third_party/rust/neqo-http3/Cargo.toml
+++ b/third_party/rust/neqo-http3/Cargo.toml
@@ -13,3 +13,3 @@
 edition = "2024"
-rust-version = "1.90.0"
+rust-version = "1.88.0"
 name = "neqo-http3"
--- a/third_party/rust/neqo-qpack/Cargo.toml
+++ b/third_party/rust/neqo-qpack/Cargo.toml
@@ -13,3 +13,3 @@
 edition = "2024"
-rust-version = "1.90.0"
+rust-version = "1.88.0"
 name = "neqo-qpack"
--- a/third_party/rust/neqo-transport/Cargo.toml
+++ b/third_party/rust/neqo-transport/Cargo.toml
@@ -13,3 +13,3 @@
 edition = "2024"
-rust-version = "1.90.0"
+rust-version = "1.88.0"
 name = "neqo-transport"
--- a/third_party/rust/neqo-udp/Cargo.toml
+++ b/third_party/rust/neqo-udp/Cargo.toml
@@ -13,3 +13,3 @@
 edition = "2024"
-rust-version = "1.90.0"
+rust-version = "1.88.0"
 name = "neqo-udp"
